src/libYARP_logger/src/YarpLogger.cpp

```diff
@@ -689,7 +690,6 @@ void LoggerEngine::set_listen_option               (std::string   option, bool e
 
 bool LoggerEngine::get_listen_option               (std::string   option)
 {
-    if (log_updater == NULL) return false;
     return false;
```

Comment:
This looks weird, are we sure that this was not supposed to be a return true? CC @randaz81

Reply:
I thought that too, but then I looked at other `get_listen_option` functions, for example https://github.com/robotology/yarp/blob/f158a2436c8897a81ba599071f3c5f9c366f8d7e/src/libYARP_logger/src/YarpLogger.cpp#L670 and this was my best guess.